azure-pipelines.yml

```diff
@@ -202,6 +202,15 @@ jobs:
     parameters:
       trigger_sha: '$(trigger_sha)'
 
+- job: compat_stable_protobuf_pr
+  pool:
+    name: ubuntu_20_04
+    demands: assignment -equals default
+  steps:
+    - checkout: self
+      persistCredentials: true
```

Comment:
```suggestion
```
I don't see where you need credentials here, and I'd rather not have them if they're not useful.